src/legacy/core_plugins/dashboard_embeddable_container/public/css/index.scss

```diff
@@ -3,4 +3,6 @@
 
 // TODO: uncomment once the duplicate styles are removed from the dashboard app itself.
 // MUST STAY AT THE BOTTOM BECAUSE OF DARK THEME IMPORTS
-@import './embeddable/index';
+// @import '../dashboard_grid';
```

Comment:
I'm trying to keep up with the shuffling of all these files, but am struggling to. Can someone please describe how these files are being imported now and what they affect?

Reply:
I uncommented all of them.

These files are not in `./np_ready` folder but in `./css` folder instead because we currently don't have SASS support in the New Platform; and in `np_ready` folder there is only stuff that we can copy-paste to the New Platform plugin.